<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>nxp-uuu-qt-tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/Resources/icon/main.ico</normaloff>:/icon/Resources/icon/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="deviceGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QLineEdit" name="chipLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>133</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="chipLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>chip</string>
     </property>
    </widget>
    <widget class="QLabel" name="fastbootLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fastboot</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fastbootPathLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>52</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="fastbootPathPushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/Resources/img/folder.png</normaloff>:/img/Resources/img/folder.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="fastbootBurnPushButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>28</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/img/Resources/img/download.png</normaloff>:/img/Resources/img/download.png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="flashMemoryComboBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>25</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>EMMC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SDCARD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NANDFLASH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORFLASH</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="flashMemoryLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>25</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash Memory</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>status</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusCurLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No connect device!</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="normalGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Normal</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QLineEdit" name="deviceTreePathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rootfsLabel">
       <property name="text">
        <string>rootfs</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="kernelPathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="kernelLabel">
       <property name="text">
        <string>kernel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="rootfsPathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="kernelPathPushButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/folder.png</normaloff>:/img/Resources/img/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="ubootBurnPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/download.png</normaloff>:/img/Resources/img/download.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="rootfsBurnPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/download.png</normaloff>:/img/Resources/img/download.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="rootfsPathPushButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/folder.png</normaloff>:/img/Resources/img/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ubootPathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="deviceTreePathPushButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/folder.png</normaloff>:/img/Resources/img/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="deviceTreeLabel">
       <property name="text">
        <string>device tree</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="kernelBurnPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/download.png</normaloff>:/img/Resources/img/download.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ubootPathPushButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/folder.png</normaloff>:/img/Resources/img/folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ubootLabel">
       <property name="text">
        <string>uboot</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="deviceTreeBurnPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/img/Resources/img/download.png</normaloff>:/img/Resources/img/download.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="deviceGroupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Advanced</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>381</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="logPlainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="clearLogPushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
